Libraries/Core/setUpBatchedBridge.js

```diff
@@ -22,7 +22,7 @@ BatchedBridge.registerLazyCallableModule('JSTimers', () =>
   require('./Timers/JSTimers'),
 );
 BatchedBridge.registerLazyCallableModule('HeapCapture', () =>
-  require('../Utilities/HeapCapture'),
+  require('../HeapCapture/HeapCapture'),
```

Comment:
requiring like this means you should also grab `default` i think.

Reply:
i'm not sure if there should be default since export looks like this:

`module.exports = HeapCapture;`

Reply:
Ah gotcha, thought this was coming direct from NativeHeapCapture